v2/cmd/integration-test/http.go

```diff
@@ -505,6 +506,31 @@ func (h *httpRawGet) Execute(filePath string) error {
 	return expectResultsCount(results, 1)
 }
 
+type httpRawPath struct{}
+
+func (h *httpRawPath) Execute(filepath string) error {
+	router := httprouter.New()
+	var routerErr error
+
+	router.GET("/", func(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+		if r.RequestURI != "/test/..;/..;/" {
```

Comment:
Similarly to https://github.com/projectdiscovery/nuclei/blob/16d15e216c70044917d8d3180e054a019ed430d4/v2/cmd/integration-test/http.go#L468 I think we can return some expected message and match it with template matchers.